utility/src/main/java/org/oppia/util/parser/ImageLoader.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.SimpleTarget
+import androidx.annotation.DrawableRes
+
+/** Helper that handles loading and caching images from remote URLs.*/
```

Comment:
This isn't really a helper, it's more of an interface describing what an image loader should be able to do. Consider:

```kotlin
/** A loader for downloading and caching images from remote URLs. */
```

Reply:
corrected.